src/ocamlorg_frontend/package_intf.ml

```diff
@@ -1,16 +1,29 @@
+type version = Latest | Version of string
+
 type package = {
   name : string;
   description : string;
   license : string;
-  version : string;
+  version : string option;
+      (* None = we are looking at the latest version of the package on a
+         /latest-URL, Some v = we are looking at version v *)
```

Comment:
Maybe using your `version` type would be more explicit/safer than a comment.

Reply:
I'll give it another try to see if I can reduce the verbosity I saw when I did this. With the helper functions it may not be too bad, and I agree that the explicitness is helpful.